node/actors/agent_manager.go

```diff
@@ -37,18 +39,18 @@ func (mgr *agentManager) HandleInspect(from gen.PID, item ...string) map[string]
 
 func (mgr *agentManager) HandleMessage(from gen.PID, message any) error {
 	switch message {
-	case "post_init":
+	case PostInit:
 		// making subscription using MonitorEvent of the gen.Process interface
 		if _, err := mgr.MonitorEvent(InternalNatsServerReady); err != nil {
 			return err
 		}
-		mgr.Log().Info("successfully subscribed to: %s", InternalNatsServerReady)
+		mgr.Log().Info("successfully subscribed to internal nats server", slog.Any("event_name", InternalNatsServerReady))
 	}
 	return nil
 }
 
 func (mgr *agentManager) HandleEvent(event gen.MessageEvent) error {
-	mgr.Log().Info("received event %s", event.Event)
+	mgr.Log().Info("received event", slog.Any("event", event.Event))
```

Comment:
we should be consistent here. if we log event names as `event_name` (line 47 above) then we should always use that field to make log searches easier.

Reply:
I noticed this also. Will fix.